Doc: Updating README.md (#27)
vtex-apps/tab-layout: docs/README.md

      "label": "Home 2"
    }
  },
  "tab-content#home": {
    "children": [
      "tab-content.item#home1",
      "tab-content.item#home2"
    ]
  },
  "tab-content.item#home1": {
    "children": [
      "carousel#home"
    ],
    "props": {
      "tabId": "home1"
    }
  },
  "tab-content.item#home2": {
    "children": [
      "shelf#home",
      "info-card#home",
      "rich-text#question",
      "rich-text#link",
      "newsletter"
    ],
    "props": {
      "tabId": "home2"
    }
  }
}
```

> > ⚠️ *Do not forget to also declare the `tab-content.item`'s children blocks in order to properly render the tab content.*

Reviewer: ```suggestion
> ⚠️ Make sure to declare the `tab-content.item`'s children blocks in order to properly render the tab content.
```

> | `blockClass` | `string` | Block ID of your choosing to be used in [CSS customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization). | `undefined` |

Reviewer: Atualizar esse link para o do Developer Portal

https://developers.vtex.com/docs/guides/vtex-io-documentation-using-css-handles-for-store-customization

| `tabId` | `string` | Tab ID of your choosing. It will be used to match the content to a given tab (defined by the `tab-list.item` / `tab-list.item.children`  blocks). | `undefined` |

> > ⚠️ *Pay attention to the chosen tab ID declared in both `tab-list.item` and `tab-content.item` blocks, once it is the key to link a given tab to its content.*

Reviewer: ```suggestion
> ⚠️ Be mindful of the `tabID` specified in both `tab-list.item` and `tab-content.item` blocks, as this parameter couples a tab and its corresponding content.
```
